src/runtime/nitro-plugin.ts

```diff
@@ -0,0 +1,9 @@
+import type { NitroAppPlugin } from 'nitropack'
+
+import { script } from '#color-mode-options'
+
+export default <NitroAppPlugin> function (nitro) {
+  nitro.hooks.hook('render:html', (htmlContext) => {
+    htmlContext.head.push(`<script>${script}</script>`)
```

Comment:
Small perf: string does needs to be generated once (could be inside function body too but out of hook